rose/client/world.py

```diff
@@ -35,7 +36,8 @@ def get(self, pos):
             Arguments:
               pos: 2 tuple (x, y) using game logical units
 
-            Accessing a position out of the world bounds will raise IndexError exception.
+            Accessing a position out of the world bounds
+            will raise IndexError exception.
```

Comment:
This works, bu we should have consistent wrapping. Should be wrapped at 80 chars - break after IndexError